docs/docs/guides/integrate/back-channel-logout.mdx

```diff
@@ -0,0 +1,177 @@
+4. The user will have to authenticate using the demanded auth mechanics.
+5. Your application will be called on the registered callback path (redirect_uri) for the authorization code exchange.
+See [OIDC Flow](/docs/guides/integrate/login/oidc/login-users) for more details.
+On successful exchange, an SSO session will be created.
+6. If the user opens another application,
+7. the application will also create an authorization request to the authorization endpoint.
+8. ZITADEL can then reuse the existing SSO session and will not ask the user to authenticate again and directly return the code for exchange.
+See [OIDC Flow](/docs/guides/integrate/login/oidc/login-users) again for details.
+9. At a later point, the user signs out from one of the applications, in this case the second one.
+10. The application will redirect the user to the end_session endpoint.
+11. ZITADEL will terminate the SSO session and redirect the user back to the application's post_logout_redirect_uri.
+The application can delete the local session.
+12. ZITADEL will also send a back-channel logout request to every registered application with previously opened sessions.
+The application can then invalidate the user's session without the need for an active browser session.
+
+### Indicating Support
+
+As required by the [specification](https://openid.net/specs/openid-connect-backchannel-1_0.html#BCSupport), ZITADEL
+will advertise the `backchannel_logout_supported` and `backchannel_logout_session_supported`
+on the discovery endpoint once the feature flag is enabled.
+The latter boolean indicates, that ZITADEL will provide a session ID (`sid`) claim as part of the logout token. This
+provides the possibility to match the exact SSO session, which was terminated.
+
+### Client
+
+To enable the back-channel logout on a client, they simply need to register a `backchannel_logout_uri` as part of their
+configuration, e.g. [creating an OIDC application](https://zitadel.com/docs/apis/resources/mgmt/management-service-add-oidc-app).
+
+As soon as the URI is set, every new authorization request will register a back-channel notification to be sent once
+the session is terminated by a user sign out.
+
+## Back-Channel Request
+
+When the session is terminated, ZITADEL will send back-channel logout requests asynchronously to every registered
+client of the corresponding session.
+The request is an `application/x-www-form-urlencoded` POST request to the registered URI including a `logout_token`
+parameter in the body.
+
+Please be aware that body *may* contain other values in addition to logout_token. Values that are not understood by the
+implementation *must* be ignored.
+
+### Logout Token
+
+The `logout_token` sent in the request is a JWT similar to an ID Token.
+
+:::info
+Note however, that a Logout Token will never contain a `nonce` claim, to make sure it cannot be used as an ID Token.
```

Comment:
Should we add a similar note to the reference docs/docs/apis/openidoauth/claims.md?

```
| nonce                                             | No             | No                                      | Yes, except when used as logout_token | No                                                   |

Reply:
update both the note here and the claims.md